modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.client/src/main/scala/edu/gemini/seqexec/web/client/components/NavBar.scala

```diff
@@ -35,7 +36,7 @@ object NavBar {
               IconTerminal.copy(p = IconTerminal.p.copy(link = true, circular = true, onClick = Callback {SeqexecCircuit.dispatch(ToggleDevConsole)}))
             )
           ),
-          SeqexecCircuit.connect(_.user)(TopMenu(_))
+          userConnect(x => TopMenu(x))
```

Comment:
`(TopMenu(_))` may work, it depends a bit if the compiler can infer the type of x, I'll try